Ghidra/Features/Decompiler/src/decompile/cpp/ifaceterm.hh

```diff
@@ -16,6 +16,9 @@
 /// \file ifaceterm.hh
 /// \brief Add some terminal capabilities to the command-line interface (IfaceStatus)
 
+#ifndef __IFACE_TERM__
```

Comment:
Shouldn't have underscore

Reply:
Hmmm. I did say the definition name was "the name of the file without the `.hh` suffix", but I also see that an existing file `ifacedecomp.hh` includes a space in the definition name https://github.com/NationalSecurityAgency/ghidra/blob/3aac7dae75b6f8cd60152086a778e1f9248ce9fb/Ghidra/Features/Decompiler/src/decompile/cpp/ifacedecomp.hh#L19-L20

It seems to me that removing the space would be best. And then I can make changes to other include guard definition names to make sure they are consistently the name of the file without `.hh` suffix.

Reply:
Looking again, there are more than a few inconsistencies with the existing guard names, like `float.hh` as a particularly interesting example---the file was maybe renamed at some point from `floatformat.hh` to `float.hh` and forgot to update the guard name? I'm not sure where `CPUI_` comes from but it might have been a subdirectory where the file used to live?

https://github.com/NationalSecurityAgency/ghidra/blob/3aac7dae75b6f8cd60152086a778e1f9248ce9fb/Ghidra/Features/Decompiler/src/decompile/cpp/float.hh#L19-L20

Reply:
@caheckman If you have a chance, would you be able to help resolve these naming inconsistencies and/or give some guidance on how you think I should proceed to make acceptable changes? Thank you.